docs/contribute-how-to.md

```diff
@@ -5,10 +5,242 @@ mentions:
 
-Well, actually this page doesn't exist at the moment, but look back later! We are working on it the moment you read this!
+This article will teach you how to use GitHub Desktop, the easiest way to manage your local Repositories. Its possible to do this without GitHub Desktop just by using Git, but this is more advanced than just using GitHub Desktop and we won't be covering this in this guide.
 
-If you have a question right now, join our [Discord Server](/discord) and ask it! We will help you.
+:::tip
+If you already have installed **VSCode**, **GitHub Desktop** and **NodeJS** and want to learn more about forking and cloning a GitHub Repository, you can continue 
+
+<BButton link='/contribute-how-to#setup-your-local-environment' color=blue>Setup local environment</BButton>
+:::
+
+:::tip
+If you already have installed the necessary software and know how to fork and clone a repository, but want to learn more about committing and Pull Requests, you can continue
+
+<BButton link='/contribute-how-to#committing-your-changes' color=blue>Committing and Pull Requests</BButton>
+:::
+
+:::tip
+If you already have installed the necessary software and know how to handle Git-Repositories and GitHub, you can continue by learning how to style our pages!
+
+<BButton link='/contribute-style' color=blue>Style our pages</BButton>
+:::
+
+## Steps to edit the wiki
+
+To edit the wiki, there are 5 major steps that you need to follow (downloading and setting up the software is just done the first time):
+
+:::tip
+The 5 steps that you need to follow to successfully update our wiki
+
+1. Update your fork to match the latest Version of our Wiki
+2. Fetch the latest changes in GitHub Desktop
+3. Edit the Wiki
+4. Commit your local changes and sync with GitHub
+5. Create a Pull Request
+:::
+
+If you see this the first time, it might look a little overwhelming, but don't worry! Its very easy and we guide you through this step by step.
+
+# Setting up the Software
+
+To edit our wiki, you need to download and install the following software:
+-   GitHub Desktop
+-   NodeJS
+-   VisualStudioCode
+
+If you wan't to edit the wiki more frequently, you might want to start using Git, as its more powerful than GitHub Desktop. But don't worry, GitHub Desktop is enough for us!
+
+## Visual Studio Code
+
+Visual Studio Code (VSCode) is a great environment to edit all your files, code something, etc. You can use other editors too, but in this tutorial we will guide you trough the installation of VSCode.
+
+#### Download
+
+Visit https://code.visualstudio.com/Download and download the Option thats best suited for you.
+
+![](/assets/images/contribute/setting_up_software/vscode/vsc_download.png)
+
+#### Installation
+
+Go to your folder where you saved the .exe and double-click it. The setup window will open. You can click on `next` most of the time, just make sure on the "additional tasks" window, that `add to PATH` is selected.
```

Comment:
> Maybe we could have an image here?

I could add this image maybe in another version (if it is not that urgent) because i would have to uninstall vscode and install it again on my test pc and i can't do this rn. 

> Also kinda weird that we describe installing VSCode here clearer than in our actual addon docs lol!

Is this a problem xD